src/util/json.h

```diff
@@ -28,6 +31,24 @@ Convenience header for Nlohmann::Json that sets the default options. Also
 #include "util/Exception.h"
 #include "util/SourceLocation.h"
 
+/*
+@brief Read the specified json file and build a json object out of it.
+
+@param jsonFileName Name of the file, or path to the file. Must describe a
+`.json` file.
+*/
+inline nlohmann::json fileToJson(std::string_view jsonFileName) {
+  // Check, if the filename/-path ends with ".json". Checking, if it's a valid
+  // file, is done by `ad_utility::makeIfstream`.
+  if (!jsonFileName.ends_with(".json")) {
+    throw std::runtime_error(absl::StrCat("The given filename/-path '",
+                                          jsonFileName,
+                                          "' doesn't describe a json file."));
```

Comment:
Explicitly state in the error, that the filename must end with `.json`, because otherwise one might think that this is already a paring error.

Reply:
I tried a new version: [bc656dc](https://github.com/ad-freiburg/qlever/pull/1024/commits/bc656dc2f95abf7a187e050a2eb90f9349f33ed3)

Is this better?